scripts/config.py

```diff
@@ -200,18 +201,26 @@ def realfull_adapter(_name, active, section):
     'MBEDTLS_ZLIB_SUPPORT', # build dependency (libz)
 ])
 
+def is_seamless_alt(name):
+    """Include xxx_ALT symbols that don't have external dependencies.
+
+    Include alternative implementations of platform functions, which  are
+    configurable function pointers that default to the built-in function.
+    This way we test that the function pointers exist and build correctly
+    without changing the behavior, and tests can verify that the function
+    pointers are used by modifying those pointers.
+
+    Exclude alternative implementations of library functions since they require
+    an implementation of the relevant functions and an xxx_alt.h header.
+    """
+    return name.startswith('MBEDTLS_PLATFORM_')
```

Comment:
I think it would be a bit cleaner to exclude `MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT` here rather than in `EXCLUDE_FROM_FULL`. Otherwise this function's name and docstring are promising more than the function actually does.

Reply:
I hesitated and came to the opposite conclusion: that it's cleaner to have a single list for ad hoc stuff. But I'll tweak the documentation.

Reply:
I'm not convinced, because it still makes the behaviour of `is_seamless_alt()` more complex to describe, and make the code less local, but your version is good enough that I don't think this difference of views is worth debating any further.